### ISSUE TITLE
확인을 부탁할 때 

### ISSUE THREAD

**Issue by Seohareyung** [Seohareyung/mini-project-Repository#5]
5/2 오후 6시54분 헤더파일 확인 부탁드립니다!

**Comment by LimJeonghyun** [comment]
네 확인했습니다! 필요한 부분 함수 네개 정도 빠진 것 같아서 메뉴 출력, 파일 저장, 출력, 학생 리스트 출력 추가하고 구조체 변수 이름 통일 했습니다!

**Comment by Seohareyung** [comment]
감사합니다 ! 확인 하겠습니다!

**Comment by Seohareyung** [comment]
위키 작성 했습니다! 로고도 한 번 만들어봤어요!ㅎㅎ

**Comment by LimJeonghyun** [comment]
파일 추가랑 함수 추가했습니다!
student.c 파일이랑 main.c파일 추가했고 student.c 파일 내에 updateStuent 함수와 delete 함수 추가해두었습니다.

**Comment by LimJeonghyun** [comment]
Updated student.c

**Comment by LimJeonghyun** [comment]
Updated student.c

**Comment by LimJeonghyun** [comment]
Updated student.c which occurs error

**Comment by Seohareyung** [comment]
정현님 피드백 확인하고 수정했는데도 계속 오류가 뜨네요ㅜ 혹시 체크 한 번만 부탁드려도 될까요?

**Comment by LimJeonghyun** [comment]
네~!! 오류 수정했습니다 :) WIKI 작성만 해주시면 될 것 같아요~

**Comment by Seohareyung** [comment]
WIKI 작성 완료 했는데, 혹시 피드백 하실 거 있나용?!

**Comment by Seohareyung** [comment]
큐ㅠ 누가 oss톡방에 팀플 이번주까지냐고 물어봐서 두근거리는 마음으로 들어갔는데, 정현님 아니여서 안도했어요🥺 저 더욱 열심히 할게요,,

**Comment by LimJeonghyun** [comment]
오 수고하셨어요~! 정확하지는 않은데 모든 함수를 다 설명하라고 하셔서 selectMenu(), selectDataNo함수, readStudent 설명도 추가해야할 것 같아요....!

**Comment by Seohareyung** [comment]
설명 추가 했습니다:)!

**Comment by LimJeonghyun** [comment]
확인했습니다!!!

**Comment by Seohareyung** [comment]
-위키 홈화면에 페이지 소개 넣어봤어요! 

"저희의 프로젝트를 소개합니다

저희의 프로젝트에 대한 설명이 궁금하다면?🤔 =>프로젝트
저희의 협업과정이 궁금하다면?🤔 =>협업 과정
 저희의 프로젝트의 디자인이 궁금하다면?👩🏻‍🎨 =>DESIGN
저희의 프로젝트에대한 코드 설명이 궁금하다면?🤔 =>MD
저희가 누군지 궁금하다면?❤️ =>WHO WE ARE "

- 더 귀여운 것 같아요 ㅎㅎ

**Comment by Seohareyung** [comment]
개발일지는 MD에 결과물이랑 설명있어서 링크 걸어두고 간단하게 끝냈어요!

**Comment by LimJeonghyun** [comment]
네! 확인했습니다.

**Comment by LimJeonghyun** [comment]
검색기능 3가지 모두 구현하고 컴파일하여 테스트까지 완료했습니다!! 검색기능 구현에 맞추어 main.c 내용도 모두 업데이트 하였습니다.
파일 저장에 필요한 student.txt 파일도 미리 만들어 두었습니다!

**Comment by Seohareyung** [comment]
파일 기능 2가지 모두 구현하고 컴파일하여 테스트 완료 했습니당! 스크린샷도 올려뒀어요!! 위키도 정리해서 업데이트 해 뒀습니다! 좋은 주말 되세요!

**Comment by Seohareyung** [comment]
확인 후에 혹시 빠진 부분 있으면 말씀해주세요! 바로 수정할게요!! 🥺🥺

**Comment by LimJeonghyun** [comment]
늦은 확인 죄송합니다..ㅠㅠ 함수 네이밍 부분 스펠링 오류 확인하고 모두 수정해두었습니다!

**Comment by Seohareyung** [comment]
네네! 저도 위키에 마지막 구현일지 수정해서 업로드 해 뒀습니다! 수고하셨어요!

**Comment by Seohareyung** [comment]
printf("\n출석? : ("yes:1 no:0):");이 부분이요! 여기서 NO누르면 바로 지각으로 가서 printf("\n번호로 입력하세요 : (출석:1, 지각:0): ");  이렇게 바꿨어요! 괜찮은가용??

**Comment by Seohareyung** [comment]
printf 할 때 (학번,이름, 출석, 지각) 부분이 띄어쓰기 안 맞는 부분있어서 맞춰놨습니다~
